contracts/stellar-example/src/tests/test.rs

```diff
@@ -1,58 +1,40 @@
 use soroban_sdk::testutils::{Address as _, AuthorizedFunction, AuthorizedInvocation};
 use soroban_sdk::token::{self, StellarAssetClient};
 use soroban_sdk::{vec, Address, Bytes, BytesN, Env, IntoVal, String, Symbol};
-use soroban_token_sdk::metadata::TokenMetadata;
-use stellar_axelar_gas_service::{AxelarGasService, AxelarGasServiceClient};
+use stellar_axelar_gas_service::testutils::setup_gas_service;
+use stellar_axelar_gas_service::AxelarGasServiceClient;
 use stellar_axelar_gateway::event::{ContractCalledEvent, MessageApprovedEvent};
-use stellar_axelar_gateway::testutils::{self, generate_proof, get_approve_hash, TestSignerSet};
+use stellar_axelar_gateway::testutils::{
+    generate_proof, get_approve_hash, setup_gateway, TestSignerSet,
+};
 use stellar_axelar_gateway::types::Message;
 use stellar_axelar_gateway::AxelarGatewayClient;
 use stellar_axelar_std::address::AddressExt;
 use stellar_axelar_std::traits::BytesExt;
 use stellar_axelar_std::types::Token;
 use stellar_axelar_std::{auth_invocation, events};
 use stellar_interchain_token_service::event::TrustedChainSetEvent;
-use stellar_interchain_token_service::testutils::INTERCHAIN_TOKEN_WASM_HASH;
-use stellar_interchain_token_service::{InterchainTokenService, InterchainTokenServiceClient};
+use stellar_interchain_token_service::testutils::{setup_its, setup_its_token};
+use stellar_interchain_token_service::InterchainTokenServiceClient;
 
 use crate::event::{ExecutedEvent, TokenReceivedEvent};
 use crate::{Example, ExampleClient};
 
-const ITS_HUB_ADDRESS: &str = "hub_address";
 const SOURCE_CHAIN_NAME: &str = "source";
 const DESTINATION_CHAIN_NAME: &str = "destination";
 
-fn setup_gateway<'a>(env: &Env) -> (TestSignerSet, AxelarGatewayClient<'a>) {
-    let (signers, client) = testutils::setup_gateway(env, 0, 5);
-    (signers, client)
-}
-
-fn setup_gas_service<'a>(env: &Env) -> AxelarGasServiceClient<'a> {
-    let owner: Address = Address::generate(env);
-    let gas_collector: Address = Address::generate(env);
-    let gas_service = env.register(AxelarGasService, (&owner, &gas_collector));
-    let gas_service_client = AxelarGasServiceClient::new(env, &gas_service);
-
-    gas_service_client
-}
-
-struct TestConfig<'a> {
+struct ExampleTestConfig<'a> {
```

Comment:
```suggestion
struct TestConfig<'a> {
```
Just keep it as test config? Example prefix is implied

Reply:
Mm ok yes makes sense